examples/angular/src/BUILD.bazel

```diff
@@ -43,6 +45,13 @@ ts_library(
     ],
 )
 
+filegroup(
```

Comment:
needs a comment about why to do this?
Also it's pretty gross that it ends up in user code. Maybe we should just use `tsc()` from `@npm//typescript:index.bzl` to transpile the initialize_testbed? but then we're further from the g3 approach

Reply:
hmmmm. I think this change was only required with the JSNamedModuleInfo as that bring in all the transitives. I can remove it from here and kick the ball a bit further on this.